packages/genesys-spark-components/src/components/stable/gux-tabs/gux-tab-list/gux-tab-list.scss

```diff
@@ -76,37 +76,11 @@ gux-tabs {
 
   .gux-scroll-button-container {
     display: flex;
```

Comment:
There is just one minor issue if you hover hover the `gux-button` it seems to be smaller than this div container that is wrapping it.